yarn.lock

```diff
@@ -2436,7 +2435,7 @@
     to-fast-properties "^2.0.0"
 
 "@backstage/catalog-model@^0.2.0":
-  version "0.6.0"
+  version "0.6.1"
```

Comment:
this and the others below should be removed from the pr

Reply:
Not sure why these aren't updated when running yarn install tbh, but should be fine to apply these here